renovate.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
What is renovate being used for here?

Reply:
Eh, yeah.... No, it isn't It was included in the template and I kinda didn't notice it there so it slipped.